File: async-scala-client/src/main/scala/io/swagger/client/model/TaxByTypeSummaryJurisdictionForGoods.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class TaxByTypeSummaryJurisdictionForGoods (
  jurisdictionName: Option[String],  // Jurisdiction used for calctax amount
jurisdictionType: Option[String],  // Type of jurisdiction - &#39;City&#39; - &#39;State&#39; - &#39;Country&#39; 
tax: Option[Double]  // sum of referenced tax value by jurisdiction
)

File: javascript-closure-angular-client/API/Client/TaxByTypeSummaryJurisdictionForGoods.js
goog.provide('API.Client.TaxByTypeSummaryJurisdictionForGoods');

/**
 * @record
 */
API.Client.TaxByTypeSummaryJurisdictionForGoods = function() {}

/**
 * Jurisdiction used for calctax amount
 * @type {!string}
 * @export
 */
API.Client.TaxByTypeSummaryJurisdictionForGoods.prototype.jurisdictionName;

/**
 * Type of jurisdiction - 'City' - 'State' - 'Country' 
 * @type {!string}
 * @export
 */
API.Client.TaxByTypeSummaryJurisdictionForGoods.prototype.jurisdictionType;

/**
 * sum of referenced tax value by jurisdiction
 * @type {!number}
 * @export
 */
API.Client.TaxByTypeSummaryJurisdictionForGoods.prototype.tax;

/** @enum {string} */
API.Client.TaxByTypeSummaryJurisdictionForGoods.JurisdictionTypeEnum = { 
  City: 'City',
  State: 'State',
  Country: 'Country',
}
